README.md

```diff
@@ -21,6 +21,43 @@ There are two possible paths, depending on your choice of Kubernetes:
 1. For any Kubernetes, follow the [installation instructions][install].
 1. On MicroK8s, you simply have to enable the [Kubeflow add-on][addon].
 
+## Testing
+
+To deploy this bundle and run tests locally, do the following:
+
+1. Set up Kubernetes, Juju, and deploy the bundle you're interested in (`kubeflow` or
+   `kubeflow-lite`) using the [installation guide](https://charmed-kubeflow.io/docs/install/). This
+   must include populating the `.kube/config` file with your Kubernetes cluster as the active
+   context
+1. Install test prerequisites:
+
+```bash
+   sudo snap install juju-wait --classic
+   sudo snap install juju-kubectl --classic
+   sudo snap install charmcraft --classic
+   sudo apt update
+   sudo apt install -y firefox-geckodriver
```

Comment:
We should add `libssl-dev` here too.

Reply:
After adding this dependency I was able to run the tests successfully @ca-scribner can you add it to the README? After that this should be good to merge

Reply:
```suggestion
   sudo apt install -y libssl-dev firefox-geckodriver
```